README.md

```diff
@@ -1,2 +1,15 @@
-# collaborative-pixel-art
-A terrible idea that was shamelessly stolen from Reddit
+# Collaborative Pixel Art
+### A terrible idea that was shamelessly stolen from Reddit
+
+We thought it was unfair that [r/place](https://www.reddit.com/r/place/) is only up for a few days at a time, so we are in the process of creating our own. 
+
+## Contributors
+[Ethan Onweller](https://github.com/ethanonweller)
+
+## TODO
+- [ ] User authentication
+- [ ] Incorporate DB and ORM
+- [ ] Frontend design
+- [ ] Zoom/Scaling
+
+...etc
```

Comment:
This part can probably be replaced by GitHub projects. Let me know in the group chat if you want to start using that.